webapp/src/components/shareBoard/shareBoard.tsx

```diff
@@ -360,7 +360,13 @@ export default function ShareBoardDialog(props: Props): JSX.Element {
                             value={selectedUser}
                             className={'userSearchInput'}
                             cacheOptions={true}
-                            filterOption={(o) => !members[o.value]}
+                            filterOption={(o) => {
+                                // render non-explicit members
+                                if (members[o.value])
+                                    return members[o.value].synthetic
```

Comment:
This style will cause lint errors, once that PR gets merged. 
```suggestion
                                if (members[o.value]) {
                                    return members[o.value].synthetic
                                }
```